wfe2/wfe.go

```diff
@@ -82,11 +82,12 @@ var errIncompleteGRPCResponse = errors.New("incomplete gRPC response message")
 // plus a few other data items used in ACME.  Its methods are primarily handlers
 // for HTTPS requests for the various ACME functions.
 type WebFrontEndImpl struct {
-	RA    rapb.RegistrationAuthorityClient
-	SA    sapb.StorageAuthorityGetterClient
-	log   blog.Logger
-	clk   clock.Clock
-	stats wfe2Stats
+	ra            rapb.RegistrationAuthorityClient
+	sa            sapb.StorageAuthorityGetterClient
+	AccountGetter AccountGetter
```

Comment:
I don't see anything directly accessing this field that requires it to be exported.